package.json

```
@@ -90,10 +90,12 @@
     "@types/rewire": "2.5.28",
     "@types/semver": "7.3.13",
     "@types/sinon": "10.0.13",
+    "@types/ungap__structured-clone": "^0.3.0",
     "@types/uuid": "9.0.1",
     "@types/vscode": "1.67.0",
     "@typescript-eslint/eslint-plugin": "5.57.0",
     "@typescript-eslint/parser": "5.57.1",
+    "@ungap/structured-clone": "^1.0.2",
```

```suggestion
    "@types/ungap__structured-clone": "0.3.0",
    "@types/uuid": "9.0.1",
    "@types/vscode": "1.67.0",
    "@typescript-eslint/eslint-plugin": "5.57.0",
    "@typescript-eslint/parser": "5.57.1",
    "@ungap/structured-clone": "1.0.2",
```

Remove the `^` specifier so the version is exact as we use Dependabot to update our specified dependencies.

---

Done 1bcd565245da14e6ca1d024b16d7ee0fa09a0411